integrations/ollama/src/haystack_integrations/components/generators/ollama/chat/chat_generator.py

```diff
@@ -247,6 +259,7 @@ def run(
             stream=stream,
             keep_alive=self.keep_alive,
             options=generation_kwargs,
+            format=self.response_format,
```

Comment:
Could you please manually test how `response_format` plays with streaming and (if it works correctly) perhaps add a test for this use case?